src/accounts.cs

```diff
@@ -112,6 +113,12 @@ interface ACAccountStore {
 		[Wrap ("RequestAccess (accountType, options == null ? null : options.Dictionary, completion)")]
 		[Async]
 		void RequestAccess (ACAccountType accountType, [NullAllowed] AccountStoreOptions options, ACRequestCompletionHandler completion);
+
+		// For some reason this was never added before, xtro caught it.
```

Comment:
We had nothing missing for iOS in xtro...
If it was in a data file it should be removed